editor/state.js

```diff
@@ -34,35 +40,49 @@ export const blocks = combineReducers( {
 		switch ( action.type ) {
 			case 'REPLACE_BLOCKS':
 				return action.blockNodes.map( ( { uid } ) => uid );
+			case 'INSERT_BLOCK':
```

Comment:
I think this action can be extended. For example, in #409 I'm adding an option "after" property to this action to insert after a specific block.